app/js/lib/view_handler/attribute_value_handler.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var objectAssign = require( 'object-assign' ),
+	_ = require( 'underscore' ),
+
+	AttributeValueHandler = {
+		'post-code': null,
+		city: null,
+		email: null,
+
+		update: function ( elements ) {
```

Comment:
I found this parameter name confusing because you aren't passing in actual elements but attribute data for elements. Maybe rename it to `elementAttributeData`?